src/protocol.cpp

```diff
@@ -58,8 +59,6 @@ static const char* ppszTypeName[] =
     // BUIP010 Xtreme Thinblocks - begin section
     NetMsgType::THINBLOCK,
     NetMsgType::XTHINBLOCK,
-    NetMsgType::XBLOCKTX,
-    NetMsgType::GET_XBLOCKTX,
```

Comment:
Why this 2 msg types have been removed?

Reply:
it's not very clear but these are inventory types which are not used .... they should have never been here.